app/policies/admin_user_policy.rb

```diff
@@ -0,0 +1,53 @@
+class AdminUserPolicy < ApplicationPolicy
+  USER_SEARCH_ROLES = %w(superadmin policy_and_abuse open_doors support tag_wrangling).freeze
+  USER_ACTION_ROLES = %w(superadmin policy_and_abuse).freeze
+
+  def index?
+    can_search_users?
```

Comment:
Should we prefer defining methods that call `can_search_users?`, or creating aliases for `can_search_users?` like in `AdminPostPolicy`?